CHANGELOG.md

```diff
@@ -43,10 +43,10 @@
    a while to receive a pong from them. Further, we avoid sending too many
    gossip messages between pings to ensure we should always receive pongs in a
    timely manner (#1137).
- * If a payment is sent, creating an outbound HTLC and sending it to our
+ * If a payment was sent, creating an outbound HTLC and sending it to our
    counterparty (implying the `ChannelMonitor` was persisted on disk), but the
    `ChannelManager` was not persisted prior to shutdown/crash, no
-   `Event::PaymentPathFailed` event was be generated if the HTLC is eventually
+   `Event::PaymentPathFailed` event was generated if the HTLC is eventually
```

Comment:
Also, "is" here could be dropped or changed to "was"

Reply:
I think I beat you to this one :)

Reply:
lol, my original comment was actually about the "is" in "HTLC is"... I missed the first "is". 😛